.dotfiles_meta/Brewfile

```diff
@@ -10,15 +10,17 @@ brew "bash"
 brew "bat"
 brew "blueutil"
 brew "diff-so-fancy"
+brew "exa"
 brew "fd"
 brew "fish"
 brew "fzf"
 brew "git"
 brew "hub"
 brew "neovim"
+brew "nnn"
 brew "ripgrep"
 brew "starship"
-brew "tree"
+brew "viu"
```

Comment:
Here are the image plugins supported by preview-tui:
> #   - optional: ueberzug, kitty terminal, viu or catimg for images
ueberzug is for X11, I am using iTerm not kitty, catimg displayed image in a very grainy and pixelated manner, so viu was my top choice.